lib/process-utils.js

```diff
@@ -0,0 +1,14 @@
+/** @babel */
+
+import * as ChildProcess from "child_process";
+import {Observable} from "rx";
+
+export const exec = Observable.fromNodeCallback(ChildProcess.exec);
+const parseCSVLine = line => line.substr(1, line.length - 2).split('","');
+
+export function runningTasks() {
+    // TODO: Implement for non-Windows platforms.
+    return exec('tasklist /FO CSV /NH')
```

Comment:
@niklasfrykholm: This is Windows only. Initially I tried using the `ps-node` package (https://www.npmjs.com/package/ps-node), but I couldn't get it to work.